component/xload/data_manager.go

```diff
@@ -54,19 +54,24 @@ type remoteDataManager struct {
 	dataManager
 }
 
-func newRemoteDataManager(remote internal.Component, statsMgr *StatsManager) (*remoteDataManager, error) {
-	log.Debug("data_manager::NewRemoteDataManager : create new remote data manager")
+type newRemoteDataManagerOptions struct {
```

Comment:
Why structure name is newRemoteDataManagerOptions ?? It shall be just RemoteDataManagerOptions

Reply:
Done